messages/sfdxurl.store.md

```diff
@@ -18,10 +18,16 @@ NOTE: The "<%= config.bin %> org display --verbose" command displays the refresh
 
 You can also create a JSON file that has a top-level property named sfdxAuthUrl whose value is the authorization URL. Finally, you can create a normal text file that includes just the URL and nothing else.
 
+Alternatively, you can pipe the SFDX authorization URL through standard input by using the --sfdx-url-stdin flag and providing the '-' character as the value.
+
 # flags.sfdx-url-file.summary
 
 Path to a file that contains the Salesforce DX authorization URL.
 
+# flags.sfdx-url-stdin.summary
+
+Provide '-' as the value to pipe the Salesforce DX authorization URL through standard input (stdin).
```

Comment:
FYI, this is what made me wonder why the flag isn't Boolean.  The summaries for flags that take a value describe what the value should be (such as "Path to a file that contains the URL" for `--sfdx-url-file`).  But in this case, it sounds like the value is always `-`, so I couldn't figure out how to write the summary correctly.  So I'm confused :)  Once I get an answer about whether or not the flag should be Boolean, I want to edit this summary.